tests/docker-compose.development.yml

```diff
@@ -0,0 +1,44 @@
+version: '2'
+services:
+  shell:
+    build: ./docker
+    depends_on:
+      - redis
+      - memcached
+      - mysql
+      - postgres
+    volumes:
+      - ..:/project
+      - ./data/config-docker.php:/project/tests/data/config.php
+    entrypoint:
+      - /bin/sh
+  phpunit:
```

Comment:
What would be the difference between `shell` and `phpunit`?

Reply:
Just the entrypoint. Phpunit runs phpunit, so yo ucan run it like `docker-compose run phpunit --list-groups`